build/pack.ps1

```diff
@@ -143,47 +138,26 @@ if ($Env:ENABLE_PYTHON -eq "false") {
     Pack-Wheel '../src/Python/'
 }
 
+# Figure out if we can run Docker or not.
+
 if ($Env:ENABLE_DOCKER -eq "false") {
     Write-Host "##vso[task.logissue type=warning;]Skipping Creating Docker Image. Env:ENABLE_DOCKER was set to 'false'."
+    $runDocker = $false;
+} elseif (("$Env:AGENT_OS" -ne "") -and ($Env:AGENT_OS.StartsWith("Win"))) {
+    Write-Host "--> Cannot create docker image on Windows."
+    $runDocker = $false;
 } else {
+    $runDocker = $true;
+}
+
+if ($runDocker) {
     Write-Host "##[info]Packing Docker image..."
     Pack-Image -RepoName "iqsharp-base" -Dockerfile '../images/iqsharp-base/Dockerfile'
 }
 
-if (($Env:ENABLE_DOCKER -eq "false") -or ($Env:ENABLE_PYTHON -eq "false")) {
+if (-not $runDocker -or ($Env:ENABLE_PYTHON -eq "false")) {
```

Comment:
for consistency, do you want to also create a variable like `$enablePython`? for use both here and above on line 133.

Reply:
My thinking here was that `$runDocker` captures both `$Env:ENABLE_DOCKER` and the requirement of a Linux host, where as the Python condition is simpler. From the perspective of minimizing changes, I wanted keep the existing logic for enabling Python given that. Happy to change for consistency, though, if you would prefer.

Reply:
No, makes sense, minimizing changes is a good reason to leave it alone.

Reply:
Sounds good, thanks!